index.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const differenceWith = require('lodash/differenceWith');
+const isMatch = require('lodash/isMatch');
+
+function isAction(value) {
+    return typeof value === 'object' && typeof value.type === 'string';
+}
+
+const wait = (store, actions, timeout) => {
+    if (!timeout) {
+        timeout = 50;
+    }
+
+    if (typeof actions !== 'function') {
+        if (!Array.isArray(actions)) {
```

Comment:
`if (typeof actions === 'string') {` // convert to array
`if (Array.isArray) {` // do map